.eslintrc.yml

```diff
@@ -26,65 +26,45 @@ rules:
       MemberExpression: 1
       SwitchCase: 1
       ImportDeclaration: first
-  max-len:
-    - error
-    - 120
-  semi:
-    - error
-    - always
-  no-unused-vars:
-    - off
-  no-constant-condition:
-    - off
+  max-len: [error, 120]
```

Comment:
I'm somewhat uninclined to accept formatting changes that don't serve any functional purpose - there's little reason to go out of the way to change the formatting here. This also strikes me as somewhat inconsistent.

Reply:
It is consistent in that any rules with no object parameters are formatted as inline JSON. But in all fairness, you could make it so that only single-word rules like `error` are going to be inlined.

It's not just a stylistic choice, it makes the eslint file much more maintainable in the long run, as more rules are added and configured. I'm planning on making a few additions, and having a list on a separate line for short things like this is just adding a lot of noise and wasting space.

Reply:
Sorry, I see what you're saying but it's too subjective. There's a very strong argument to be made that this is less maintainable as any changes to rule configuration later on may require breaking out of inline-JSON or rewriting as inline-JSON. It's better to just be consistent, and generally generous spacing is actually a good thing for readability. If compactness is needed for whatever reason, such as scanning rules that are present, editor folding is the best solution.